app/gui/src/project-view/components/ComponentBrowser/component.ts

```diff
@@ -107,11 +108,21 @@ export function makeComponent({ id, entry, match }: ComponentInfo): Component {
   }
 }
 
+const ANY_TYPE = unwrap(tryQualifiedName('Standard.Base.Any.Any'))
+
 /** Create {@link Component} list from filtered suggestions. */
 export function makeComponentList(db: SuggestionDb, filtering: Filtering): Component[] {
   function* matchSuggestions() {
+    // All types are descendants of `Any`, so we can safely prepopulate it here.
+    // This way, we will use it even when `selfArg` is not a valid qualified name.
+    const additionalSelfTypes: QualifiedName[] = [ANY_TYPE]
```

Comment:
It would be more efficient to consider the `Any` type in the filter predicate rather than add it to every list--it would also avoid the potential mistake of omitting it from the list.